sklearn/metrics/ranking.py

```diff
@@ -110,10 +102,12 @@ def auc(x, y, reorder=False):
 
 
 def average_precision_score(y_true, y_score, average="macro",
-                            sample_weight=None):
+                            sample_weight=None, interpolation=None):
     """Compute average precision (AP) from prediction scores
 
-    This score corresponds to the area under the precision-recall curve.
+    Optionally, this will compute an eleven-point interpolated average
+    precision score, which fixes recall values and averages precisions
```

Comment:
Yeah, I find it a bit hard to explain in a sentence! How about

> Optionally, this will compute an eleven-point interpolated average precision score: for 11 target recall values ([0, 0.1, 0.2, ... 1.0]), this averages the best precision values possible while meeting or exceeding the target recall.